.github/CODEOWNERS

```diff
@@ -20,9 +20,6 @@
 /src/pages/LandingPage* @DataBiosphere/terra-cobranding
 /src/pages/billing/ @DataBiosphere/broadworkspaces-terra-ui
 /src/pages/workspaces/* @DataBiosphere/broadworkspaces-terra-ui
```

Comment:
The entry at line 22 is non-recursive, so it captures our very minimal Page classes.